test/integration/main_test.go

```diff
@@ -169,19 +184,24 @@ func restartInsightsOperator(t *testing.T) {
 	t.Log(errPod)
 }
 
-func checkPodsLogs(t *testing.T, kubeClient *kubernetes.Clientset, message string) {
+func _checkPodsLogs(t *testing.T, kubeClient *kubernetes.Clientset, message string, newLogs bool) {
+	r, _ := regexp.Compile(message)
 	newPods, err := kubeClient.CoreV1().Pods("openshift-insights").List(metav1.ListOptions{})
 	if err != nil {
 		t.Fatal(err.Error())
 	}
-
+	tajm := metav1.NewTime(time.Now())
```

Comment:
tajm - is it supposed to be `time`? :)

Reply:
oh :D the last variable that made my life happier, forgot to rename it. It was time, but it's already taken, unfortunately :D